src/main/webapp/app/app.component.html

```diff
@@ -4,12 +4,13 @@
       <div class="logo">
         <img src="assets/images/logos/coyo/logo-coyo-inversed-nav-hd.png" alt="COYO">
       </div>
+      <tiny-task-search (searchEvent)="searchTask($event)"></tiny-task-search>
```

Comment:
UI wise I probably would have placed this below the header and above the listed items